src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -152,10 +152,13 @@ private static void ShowResults(UpdateResults results)
 					}
 				}
 
-				Trace.WriteLine($"{results.PropsFileUpdateResults.UnrecognizedVariables.Count} variables were not found in props files:");
-				foreach (var variable in results.PropsFileUpdateResults.UnrecognizedVariables)
+				if(results.PropsFileUpdateResults.UnrecognizedVariables.Count != 0)
 				{
-					Trace.WriteLine($"        {variable.Name} in {variable.ReferencedFilePath}");
+					Trace.WriteLine($"{results.PropsFileUpdateResults.UnrecognizedVariables.Count} variables were not found in props files:");
+					foreach (var variable in results.PropsFileUpdateResults.UnrecognizedVariables)

```

Comment:
minor:  I recall the rest of the code uses concrete variable declarations instead of `var` use.  Please consider style homogeneity.

Reply:
Fixed in multiple occurances.